src/lang/en-US.json

```diff
@@ -915,6 +915,125 @@
   "vision_all_time": "All time",
   "vision_placed_observer": "placed Observer at",
   "vision_placed_sentry": "placed Sentry at",
-  "vision_ward_log": "Ward Log"
-  
+  "vision_ward_log": "Ward Log",
+
+  "chatwheel_0": "Okay.",
+  "chatwheel_1": "Careful!",
+  "chatwheel_2": "Get Back!",
+  "chatwheel_3": "We need wards.",
+  "chatwheel_4": "Stun now!",
+  "chatwheel_5": "Help!",
+  "chatwheel_6": "Push now",
+  "chatwheel_7": "Well played!",
+  "chatwheel_8": "Missing!",
+  "chatwheel_9": "Missing top!",
+  "chatwheel_10": "Missing mid!",
+  "chatwheel_11": "Missing bottom!",
+  "chatwheel_12": "Go!",
+  "chatwheel_13": "Initiate!",
+  "chatwheel_14": "Follow me",
+  "chatwheel_15": "Group up",
+  "chatwheel_16": "Spread out",
+  "chatwheel_17": "Split up and farm",
+  "chatwheel_18": "Attack now!",
+  "chatwheel_19": "Be right back",
+  "chatwheel_20": "Dive!",
+  "chatwheel_21": "On my way",
+  "chatwheel_22": "Get ready",
+  "chatwheel_23": "Bait",
+  "chatwheel_24": "Heal",
+  "chatwheel_25": "Mana",
+  "chatwheel_26": "Out of mana",
+  "chatwheel_27": "Skills on cooldown",
+  "chatwheel_28": "Ultimate ready",
+  "chatwheel_29": "Enemy returned",
+  "chatwheel_30": "All enemy heroes missing!",
+  "chatwheel_31": "Enemy incoming!",
+  "chatwheel_32": "Invisible enemy nearby!",
+  "chatwheel_33": "Enemy has rune",
+  "chatwheel_34": "Split push",
+  "chatwheel_35": "Coming to gank",
+  "chatwheel_36": "Requesting a gank",
+  "chatwheel_37": "Fight under the tower",
+  "chatwheel_38": "Deny the tower!",
+  "chatwheel_39": "Buy a courier please",
+  "chatwheel_40": "Can anyone upgrade the courier?",
+  "chatwheel_41": "We need detection",
+  "chatwheel_42": "They have detection",
+  "chatwheel_43": "Buy a Town Portal Scroll",
+  "chatwheel_44": "Re-use courier",
+  "chatwheel_45": "Deward please",
+  "chatwheel_46": "Building Mekansm",
+  "chatwheel_47": "Building Pipe of Insight",
+  "chatwheel_48": "Stack and pull please",
+  "chatwheel_49": "Pull creeps please",
+  "chatwheel_50": "Pulling creeps",
+  "chatwheel_51": "Stack neutrals",
+  "chatwheel_52": "Jungling",
+  "chatwheel_53": "Roshan",
+  "chatwheel_54": "Affirmative",
+  "chatwheel_55": "Wait",
+  "chatwheel_56": "Pause please!",
+  "chatwheel_57": "Current Game Time",
+  "chatwheel_58": "Check runes please",
+  "chatwheel_59": "Let's smoke gank!",
+  "chatwheel_60": "Good luck, have fun",
+  "chatwheel_61": "Nice",
+  "chatwheel_62": "Thanks!",
+  "chatwheel_63": "Sorry",
+  "chatwheel_64": "Don't give up!",
+  "chatwheel_65": "That just happened",
+  "chatwheel_66": "Game is hard",
+  "chatwheel_67": "New meta",
+  "chatwheel_68": "My bad",
+  "chatwheel_69": "I immediately regret my decision",
+  "chatwheel_70": "Relax, you're doing fine",
+  "chatwheel_71": "%s1 is missing!",
```

Comment:
We might want to replace this with just "A hero is missing" or something generic